src/rustdoc_gen.rs

```diff
@@ -248,29 +249,25 @@ impl FeatureConfig {
             return String::new();
         }
 
-        let mut extra_features = self.extra_features.clone();
-        // Sort the features, because the order is not revelant
-        extra_features.sort();
-
-        let unique_identifier = format!(
-            "{}{}",
-            match self.features_group {
-                FeaturesGroup::All => "A",
-                FeaturesGroup::Default => "D",
-                FeaturesGroup::Heuristic => "H",
-                FeaturesGroup::None => "N",
-            },
-            extra_features.join("@#$"), // separate features with some unusual symbols
-        );
-
-        // Calculate a rolling hash for the identifier, as it could end up too long.
-        let mut hash = Wrapping(0_u64);
-        let base = Wrapping(263_u64); // a hand-picked prime number
-        for b in unique_identifier.as_bytes() {
-            hash = hash * base + Wrapping(*b as u64);
-        }
-
-        format!("{:x}", hash)
+        // Sort the features, because the order is not revelant.
```

Comment:
```suggestion
        // Sort the features to ensure a deterministic order.
```
"Not relevant" is at risk of communicating the exact opposite of the intended meaning. The order isn't relevant for rustdoc, but is really relevant for our hashing or else we might needlessly miss the cache a lot.